src/lib/event_mapper/impl/glue_event_mapper.py

```diff
@@ -0,0 +1,32 @@
+from .general_aws_event_mapper import GeneralAwsEventMapper
+from ...settings import Settings
+
+
+class GlueEventMapper(GeneralAwsEventMapper):
+    def __init__(self, settings: Settings):
+        super().__init__(settings)
+
+    def get_resource_name(self, event):
+        return event["detail"]["jobName"]
+
+    def get_message_body(self, event):
+        message_body = []
+        table = {}
+        rows = []
+        table["table"] = {}
+        table["table"]["rows"] = rows
+        message_body.append(table)
+
+        style = "error" if event["detail"]["severity"] == "ERROR" else None
```

Comment:
just curious - if we have State = "STOPPED" (meaning it was stopped manually, which is kindof an error) - will it still be severity == ERROR?

Also, I believe, going forward, we'd need to discuss how to distinguish notifications about successful executions and failures

Reply:
Good question, I would believe that it wouldn't have "error" severity, as it's not listed as "abnormal" run status for jobs, so only "FAILED", "ERROR" and "TIMEOUT" would be treated as something not good happening: https://docs.aws.amazon.com/glue/latest/dg/job-run-statuses.html

However we can discuss how to better determine the severity of events 👍 

Agree, this one's an initial implementation. Created a card in Trello.